src/content/docs/index.mdx

```diff
@@ -25,16 +25,34 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 ## Next steps
 
 <CardGrid stagger>
-	<Card title="Writing and Style Guides" icon="pencil">
-		Learn how to write text and code for Astro docs.
-	</Card>
-	<Card title="i18n and Translator guides" icon="add-document">
-		Add contributions to Astro docs in other languages.
-	</Card>
-	<Card title="Maintainer and Reviewer Guides" icon="setting">
-		Learn how to manage PRs in the Astro docs repo as a maintainer or reviewer.
-	</Card>
-	<Card title="Read the docs" icon="open-book">
-		Visit [Astro Docs](https://docs.astro.build/) to learn about Astro!
-	</Card>
+
+  <Card title="Writing and Style Guides" icon="pencil">
+  
+    Learn how to write text and code for Astro docs.
+
+    - [Astro Docs writing style](/guides/writing-style/)
+
+  </Card>
+
+  <Card title="i18n and Translator guides" icon="add-document">
+
+    Add contributions to Astro docs in other languages.
+
+    - [How to translate Astro’s docs](/guides/i18n/)
+    - [How to review translation PRs](/reviewers/reviewing-translations/)
+
+  </Card>
+
+  <Card title="Maintainer and Reviewer Guides" icon="setting">
```

Comment:
```suggestion
    <Card title="Add docs for your code" icon="seti:html">

    See how to add docs to accompany your Astro code PR.

    - [Documenting code changes](/docs-for-code-changes/about-documenting-code-changes/)

  </Card>

  <Card title="Maintainer and Reviewer Guides" icon="setting">
```
What would you think about adding this, since this seems to be a section I end up linking people to often?

Reply:
Fine by me! "Docs for your code" confused me a bit until I read the description a couple of times. 

Maybe something like "document changes to the Astro codebase" might be clearer?

Reply:
Sounds good!